code/modules/mob/living/carbon/human/human.dm

```diff
@@ -260,13 +260,18 @@
 	popup.open()
 
 // called when something steps onto a human
-// this could be made more general, but for now just handle mulebot
 /mob/living/carbon/human/Crossed(atom/movable/AM)
 	var/obj/machinery/bot/mulebot/MB = AM
 	if(istype(MB))
 		MB.RunOver(src)
 
-	spreadFire(AM)
+	var/mob/living/carbon/M = AM

```

Comment:
you aren't checking if it really is a carbon, it will make M.fire_stacks > 0 runtime when crossing anything that isn't a living mob.